tools/serverpod_cli/lib/src/migrations/generator.dart

```diff
@@ -204,8 +204,8 @@ class MigrationGenerator {
     if (liveDatabase.installedModules == null) {
```

Comment:
Suggestion: Retrieve the value before the null check so we don't have to opt out of null safety. This allows the compiler to always give a warning if the code is modified later.

```suggestion
    var installedModules = liveDatabase.installedModules;
    if (installedModules == null) {
```

Reply:
Fixed!